src/components/layouts/Header/Header.js

```diff
@@ -40,6 +40,11 @@ export const Header = observer(() => {
 
   const logOut = async () => {
     deactivate();
+    localStorage.removeItem('connector');
+    if (window.localStorage.getItem('connector') === 'walletconnect') {
+      await walletconnect.close();
+      walletconnect.walletConnectProvider = null;
+    }
```

Comment:
Duplicated code. I think this code should be moved to `deactivate` or a reusable function.

Reply:
I'm still doing kiss and then I'll put it in the "useAuth" hook

Reply:
thank